### PR TITLE
Rodrigo menacho/count the characters

### PR DESCRIPTION
**Pull request by EotronRM**

### CODE REVIEW

File: src/test/java/org/fundacionjala/coding/CharcountTest/CCTest.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package org.fundacionjala.coding.CharcountTest;
```
Comment (carledriss) [inline src/test/java/org/fundacionjala/coding/CharcountTest/CCTest.java]: MINOR Rename this package name to match the regular expression '^[a-z]+(\.[a-z][a-z0-9]*)*$'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00120)

File: src/main/java/org/fundacionjala/coding/Charcount/CC.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
+package org.fundacionjala.coding.Charcount;
```
Comment (carledriss) [inline src/main/java/org/fundacionjala/coding/Charcount/CC.java]: MINOR Rename this package name to match the regular expression '^[a-z]+(\.[a-z][a-z0-9]*)*$'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00120)

File: src/main/java/org/fundacionjala/coding/Charcount/CC.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    /**
+     * This function count the characters about the problem.
+     * @param str **Input String**
+     * @param c **Character to count**
+     * @return **Return the Number of the character appear**
+     */
+    public static int charCount(final String str, char c) {
+        int count = 0;
+        for (int i = 0; i < str.length(); i++) {
+            char ch = (char) str.charAt(i);
```
Comment (carledriss) [inline src/main/java/org/fundacionjala/coding/Charcount/CC.java]: MINOR Remove this unnecessary cast to "char". [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1905)

**Comment by carledriss** [comment]
SonarQube analysis reported 3 issues
* ![MINOR][MINOR] 3 minor

Watch the comments in this conversation to review them.

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by carledriss** [comment]
SonarQube analysis reported 2 issues
* ![MINOR][MINOR] 2 minor

Watch the comments in this conversation to review them.

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Review by EotronRM — COMMENTED** [review]
fix regular expressions

File: src/.idea/workspace.xml [hunk 4]
```diff
@@ -0,0 +1,617 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (carledriss) [inline src/.idea/workspace.xml:1]: Don't commit this file

**Review by carledriss — APPROVED** [review]

**Pull request merged**